forc-plugins/forc-doc/src/render.rs

```diff
@@ -1100,7 +1100,7 @@ impl Renderable for ModuleIndex {
                                 h1(class="fqn") {
                                     span(class="in-band") {
                                         : title_prefix;
-                                        a(class="module", href=IDENTITY) {
+                                        a(class="mod", href=IDENTITY) {
```

Comment:
This was a small fix, it fixes the highlighting for modules. I didn't notice it at first because all other types get highlighting.